tests/docker/run-tests.sh

```
@@ -94,38 +95,35 @@ _EOF_
+    "SELECT data FROM t4 WHERE id = 2")
+
+  if [ "$converted_row" != "trigger missing key on UPDATE" ];
   then
+	  psql -h ${peer_names[0]} -c "SELECT * FROM t4 ORDER BY id;"
 	  psql -h ${peer_names[0]} -c "select * from spock.exception_log;"
-	  echo "Did not find an exception log entry. Exiting..."
+	  echo "UPDATE of a missing row was not applied as an INSERT. Exiting..."
 	  exit 1
   fi
 
   resolution_check=$(psql -X -A -t -h ${peer_names[0]} -c \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)